Misc/NEWS.d/next/Library/2019-01-11-12-20-48.bpo-35270.Vtm5iW.rst

```diff
@@ -0,0 +1 @@
+Cmd.complete: use ``completedefault`` also with ``cmd=None`` returned from ``parseline``.
```

Comment:
Ok, will amend it - can/should I push with "[ci skip]" or something similar?

Reply:
I don't think that works. Also, please just push a new commit rather than amending the last as it would make conflicts for reviewers that pullled the code and all the branch will get squashed anyway.